doc/doc/advanced/incremental.md

```diff
@@ -42,8 +42,8 @@ You can follow the [corresponding section in the original walkthrough](../basics
 Alternatively, you can try the handy scripts included incremental example we include in the source tree.
 
 ```bash
-cd examples/spouse_example/postgres/incremental
-./0-setup.sh spouse_example.f1.ddl inc-base.out
+cd examples/spouse_example/incremental
```

Comment:
It seems your merge/rebase reverted some of my fixes on the docs.  Could you make sure your commits are just renaming .ddl -> ddlog?  Unfortunately, we can't check whether the gist-it stuff works until the PR is merged (something to be fixed using Jekyll's variables), but you can easily see the paths without `postgres/` will be incorrect.

This is why it's super important to always base your work off of origin/master to avoid merge headaches.
```bash
git remote update
git checkout -b work origin/master
```